security/pkg/stsservice/mock/mockserver.go

```diff
@@ -0,0 +1,248 @@
+// Copyright 2019 Istio Authors
```

Comment:
This file can be moved to google plugin folder.

Reply:
Done. Thanks.